### PR TITLE
build: Separate production and development environments when building

### PR DESCRIPTION
**Pull request by Jtrust**
Remove the plugin for the development environment when building for production

image

### CODE REVIEW

**Comment by wu-sheng** [comment]
@arugal Let's tag and release 0.5.1 first, then we could move on 0.6.0

**Review by Fine0830 — COMMENTED** [review]
Could you please resolve those conflicts?

**Comment by Fine0830** [comment]
Uhh...`start` and `server` are only needed for the development env. `build` is for production env.

File: package-lock.json [hunk 1]
```diff
@@ -1301,7 +1310,7 @@
     },
     "dedent": {
       "version": "0.7.0",
-      "resolved": "https://registry.npmjs.org/dedent/-/dedent-0.7.0.tgz",
+      "resolved": "https://registry.npm.taobao.org/dedent/download/dedent-0.7.0.tgz",
```
Comment (Fine0830) [inline package-lock.json:1313]: You shouldn't update the resolved url to `registry.npm.taobao.org`.

**Review by Fine0830 — CHANGES_REQUESTED** [review]

**Review by Fine0830 — APPROVED** [review]

**Pull request merged**